PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -69,31 +62,37 @@ class PayPalWebCheckoutClient internal constructor(
         analytics.notifyVaultStarted(request.setupTokenId)
         val result = payPalWebLauncher.launchPayPalWebVault(activity, request)
         when (result) {
-            is PayPalPresentAuthChallengeResult.Failure -> {
-                analytics.notifyVaultAuthChallengeFailed(request.setupTokenId)
-                vaultListener?.onPayPalWebVaultFailure(result.error)
-            }
-
             is PayPalPresentAuthChallengeResult.Success ->
                 analytics.notifyVaultAuthChallengeStarted(request.setupTokenId)
```

Comment:
I think in iOS, we launch an equivalent event before attempting to launch a web view. 
So if we wanted to track success/failure of the entire 3ds flow including the web view launch, we would have a starting point.

Reply:
Yeah I'm hoping we'll be able to align analytics on iOS and Android sometime soon in an upcoming sprint.